src/components/styled-atlaskit/Select/Select.tsx

```diff
@@ -5,22 +5,30 @@
  */
 
 import React from "react";
-import { default as AkSelect, SelectProps as AkSelectProps } from "@atlaskit/select";
+import {
+  default as AkSelect,
+  SelectProps as AkSelectProps,
+  CreatableSelect,
```

Comment:
Not sure if I should rename the import to add `Ak` prefix here. Suggestions are welcome

Reply:
we don't have to do that unless we want to avoid two variables having the same name